packages/seacas/libraries/ioss/src/cgns/Iocgns_DatabaseIO.C

```diff
@@ -1819,9 +1818,8 @@ namespace Iocgns {
 
     // Create a lambda to eliminate some duplicate code in coordinate outputs...
     auto coord_lambda = [&data, &first,
-                         base](const char *ordinate, int cgns_file_ptr,
-                               const std::vector<CGNSIntVector> &block_local_node_map,
-                               int                               myProcessor) {
+                         base, this](const char *ordinate, int cgns_file_ptr,
+                               const std::vector<CGNSIntVector> &block_local_node_map) {
```

Comment:
should `block_local_node_map` be changed to `m_blockLocalNodeMap` and accessed through `this` now also?  Also potentially `get_file_pointer()`.  

Probably something to look at later instead of on this PR.